_includes/links.md

```diff
@@ -24,6 +24,7 @@
 
 [0068]: {{site.baseurl}}/recipe/0068-newspaper/ "A newspaper"
 [0117]: {{site.baseurl}}/recipe/0117-add-image-thumbnail/ "Manifest Thumbnail"
+[0230]: {{site.baseurl}}/recipe/0230-navdate/ "Chronology with navDate"
```

Comment:
Could you update this with the new recipe title?

Reply:
Yes indeed. While I'm at it, is that a good section for this recipe? My apologies for not raising it at the last authors' call.

Reply:
Maybe leave it where it is for now but I think we need to look at the headings now we have so many done in the first section.